invenio_app_ils/records/api.py

```diff
@@ -55,31 +70,40 @@ class Item(IlsRecord):
     pid_field = "item_pid"
     _pid_type = ITEM_PID_TYPE
     _schema = "items/item-v1.0.0.json"
+    _loan_resolving_path = (
+        "{scheme}://{host}/api/resolver/circulation/items/{pid_value}/loan"
+    )
+    _internal_location_resolving_path = (
+        "{scheme}://{host}/api/resolver/internal-locations/{pid_value}"
+    )
+    _document_resolving_path = (
+        "{scheme}://{host}/api/resolver/items/document/{pid_value}"
+    )
 
     @classmethod
     def create(cls, data, id_=None, **kwargs):
         """Create Item record."""
         data["circulation_status"] = {
-            "$ref": "{scheme}://{host}/api/resolver/circulation/items/{pid_value}/loan".format(
+            "$ref": cls._loan_resolving_path.format(
```

Comment:
This is very nice, extracting it to class properties; 
Irrelevant with programming, in English language -ing indicates duration in action, I would prefer `*_resolver_path` or `*_resolve_path`.